spectroscopy/code_src/herschel_functions.py

```diff
@@ -1,45 +1,48 @@
-## Herschel PACS & SPIRE (from ESA TAP)
-from astroquery.esa.hsa import HSA
-from astroquery.exceptions import LoginError
-from requests.exceptions import ChunkedEncodingError, ConnectionError
-import tarfile 
-from astropy.io import fits
+# Herschel PACS & SPIRE (from ESA TAP)
 import glob
-import pandas as pd
+import os
+import tarfile
+
 import astropy.constants as const
+import pandas as pd
 from astropy import units as u
-import os
+from astropy.io import fits
+from astroquery.esa.hsa import HSA
+from astroquery.exceptions import LoginError
+from requests.exceptions import ChunkedEncodingError, ConnectionError
 
 from data_structures_spec import MultiIndexDFObject
 
+
 def find_max_flux_column(df):
-  """
-  Analyzes a DataFrame with flux columns and returns the column with the largest sum.
+    """
+    Analyzes a DataFrame with flux columns and returns the column with the largest sum.
+
+    Args:
```

Comment:
we should probably also run some docstring formatting, e.g. these should be numpydoc formatted like the rest.
(beyond scope for this PR)

Reply:
I'm working on the docstrings now in a separate branch. If you know of a tool that will format them, let me know.

Reply:
numpydoc comes with a linter/validator but it really lights up as a chrismass tree when I run it on the code directories here.